swift-build.sh

```diff
@@ -58,4 +56,26 @@ build() {
     echo "Finished compiling ${CMAKE_BUILD_TYPE} for ${ANDROID_ABI}"
 }
 
+readonly SCRIPT_ROOT=$(cd $(dirname $0); echo -n $PWD) # path of this file
+readonly SWIFT_ANDROID_TOOLCHAIN_PATH="${SWIFT_ANDROID_TOOLCHAIN_PATH:-$SCRIPT_ROOT}"
+
+for LAST_ARGUMENT in $@; do :; done
+readonly PROJECT_DIRECTORY=${LAST_ARGUMENT:-$PWD}
```

Comment:
Can you check whether we ever get into the fallback case here? Bash has some weird rules with this fallback syntax

Reply:
jep, works like charm